src/components/Chat/index.tsx

```diff
@@ -83,7 +84,11 @@ const mapDispatchToProps = {
   getChatlistTimestamp
 };
 
-const mapStateToProps = state => ({ auth: state.auth, chat: state.chat });
+const selector = (state) => {
+  return ({ auth: state.auth, chat: state.chat, messages: state.messages[state.chat.activeChat.chatId] });
```

Comment:
is it always resolved: state.chat.activeChat.chatId?

should we set default value for messages
like []?